rl_loop/distributed_flags

```diff
@@ -20,6 +20,10 @@
 --sgd_momentum=0.95
 --steps_to_train=1024
 --window_size=500000
+--cbt_project=tensor-go
+--cbt_instance=minigo-instance
+--cbt_table=v14-games
```

Comment:
I'm curious whether you could run 2 k8s clusters in the same GCP project - would we run into hopeless collisions everywhere?